### PR TITLE
Add anchors to PR dashboard

### PR DESCRIPTION
**Pull request by cblecker**
Small PR to add anchors that can be linked to, to the PR dashboard.

/area gubernator

### CODE REVIEW

File: gubernator/templates/pr_dashboard.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
 		% continue
 	% endif
 	<h3 class="pr-section">
+	<a id="section-{{title|lower|replace(" ", "-")}}" />
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: `{{title|slugify}}` would do the same in a more standard way

Comment (cblecker) [inline gubernator/templates/pr_dashboard.html]: This was not a thing I knew of! Fixed :)

File: gubernator/templates/pr_dashboard.html [hunk 2]
```diff
@@ -22,6 +22,7 @@
 		% continue
 	% endif
 	<h3 class="pr-section">
+	<a id="section-{{title|slugify}}" />
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: Last thing: make this <a></a>-- Closing tags like this doesn't work in HTML5 https://stackoverflow.com/a/3558200

Comment (cblecker) [inline gubernator/templates/pr_dashboard.html]: I didn't know that. I tested locally in Chrome and it worked, but I guess other browsers have different behaviours? Either way, better to play it safe and fixed it.

**Pull request merged**